vagrantfile

```diff
@@ -77,7 +79,7 @@ Vagrant.configure(VAGRANT_FILE_API_VERSION) do |config|
     config.vm.provider "hyperv" do |v, override|
       override.vm.box = "gusztavvargadr/w16s-sql17d"   # temporary box url. We'll need to build one eventually
       config.vm.network "public_network", bridge: "External Connection"
-      override.vm.synced_folder ".", "/vagrant", type: "smb", smb_host: "192.168.1.3" # this is required or it guesses the wrong IP
+      # override.vm.synced_folder ".", "/vagrant", type: "smb", smb_host: "192.168.1.3" # this is required or it guesses the wrong IP
```

Comment:
Do we want to remove this? if we do, should we kill this line?

Reply:
Seems to work OK without it, I'll kill it.